specification/monitor/resource-manager/microsoft.insights/stable/2018-04-16/scheduledQueryRule_API.json

```diff
@@ -0,0 +1,794 @@
+    },
+    "MetricTrigger": {
+      "type": "object",
+      "properties": {
+        "thresholdOperator": {
+          "$ref": "#/definitions/ConditionalOperator",
+          "description": "Evaluation operation for Metric -'GreaterThan' or 'LessThan' or 'Equal'."
+        },
+        "threshold": {
+          "format": "double",
+          "type": "number"
+        },
+        "metricTriggerType": {
+          "$ref": "#/definitions/MetricTriggerType",
+          "description": "Metric Trigger Type - 'Consecutive' or 'Total'"
+        },
+        "metricColumn": {
+          "type": "string",
+          "description": "Evaluation of metric on a particular column"
+        }
+      }
+    },
+    "ConditionalOperator": {
+      "type": "string",
+      "enum": [
+        "GreaterThan",
+        "LessThan",
+        "Equal"
+      ],
+      "x-ms-enum": {
+        "name": "ConditionalOperator",
+        "modelAsString": true
+      }
+    },
+    "MetricTriggerType": {
+      "type": "string",
+      "enum": [
+        "Consecutive",
+        "Total"
+      ],
+      "x-ms-enum": {
+        "name": "metricTriggerType",
+        "modelAsString": true
+      }
+    },
+    "AlertSeverity": {
+      "type": "string",
+      "enum": [
+        "1",
+        "2",
+        "3",
+        "4"
+      ],
+      "x-ms-enum": {
+        "name": "AlertSeverity",
+        "modelAsString": true
+      }
+    },
+    "AlertStatus": {
+      "type": "string",
+      "enum": [
+        "Active",
+        "Inactive"
+      ],
+      "x-ms-enum": {
+        "name": "AlertStatus",
+        "modelAsString": true
+      }
+    },
+    "QueryType": {
+      "type": "string",
+      "enum": [
+        "ResultCount",
+        "Number"
+      ],
+      "x-ms-enum": {
+        "name": "QueryType",
+        "modelAsString": true
+      }
+    },
+    "LogSearchRule": {
+      "properties": {
+        "description": {
+          "type": "string",
+          "description": "The description of the Log Search rule."
+        },
+        "Enabled": {
+          "type": "string",
+          "description": "The flag which indicates whether the Log Search rule is enabled. Value should be true or false",
+          "enum": [
+            "true",
+            "false"
+          ],
+          "x-ms-enum": {
+            "name": "Enabled",
+            "modelAsString": true
+          }
+        },
+        "skuType": {
+          "type": "string",
+          "enum": [
+            "L1",
+            "L2",
+            "L3"
+          ],
+          "x-ms-enum": {
+            "name": "skuType",
+            "modelAsString": true
+          },
+          "description": "SKU Type {L1, L2, L3}"
+        },
+        "lastUpdatedTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "Last time the rule was updated in IS08601 format."
+        },
+        "provisioningState": {
```

Comment:
please define as an enum and list all the possible values.

Reply:
Fixed in latest iteration